dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/imports/validation/validator/relationship/SecurityOwnershipValidatorTest.java

```diff
@@ -70,25 +73,25 @@ public void setUp() {
     TrackerIdSchemeParams idSchemes = TrackerIdSchemeParams.builder().build();
     reporter = new Reporter(idSchemes);
     relationship = new Relationship();
+    relationshipType = new RelationshipType();
+
+    when(bundle.getPreheat().getRelationshipType(any())).thenReturn(relationshipType);
+    when(bundle.getStrategy(relationship)).thenReturn(CREATE);
   }
 
   @Test
   void shouldFailWhenUserHasNoAccessToRelationshipType() {
```

Comment:
Are we covering the different test cases? I understand this validation happens when the strategy is update/delete/create. Also, I see the integration and JSON cover only the negative case, but I don't have a full vision of what is already present